src/js/components/Accordion.js

```diff
@@ -0,0 +1,69 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import List from './List';
+
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.ACCORDION;
+
+export default class Accordion extends Component {
+  constructor(props) {
+    super(props);
+    this._activatePanel = this._activatePanel.bind(this);
+
+    this.state = {
+      activeIndex: props.initialIndex
+    };
+  }
+
+  _activatePanel (index) {
+    this.setState({activeIndex: index});
+  }
+
+  render () {
+    const {
+      animate,
+      className,
+      children,
+      colorIndex,
+      openMulti
+    } = this.props;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      className
+    );
+
+    const accordionChildren = React.Children.map(children, (child, index) => {
+      return React.cloneElement(child, {
+        id: 'accordion-panel-' + index,
+        isOpen: !openMulti ? (this.state.activeIndex === index) : null,
+        onTitleClick: () => {
```

Comment:
I don't see `onTitleClick` in any propTypes. Also, can we align it with 'active' and call it `onActive`?